config.json

```diff
@@ -76,6 +80,11 @@
         "bindAddress": "localhost",
         "port": 9991
     },
+    "pfsDaemon": {
+        "bindAddress": "localhost",
+        "port": 9992,
+        "isPassthrough": true
```

Comment:
`isPassthrough` is set here but isn't used.

Reply:
Fixed